docs/pages/inboxes/push-notifications.md

```diff
@@ -0,0 +1,99 @@
+## Support push notifications
+
+:::warning[🚧 XMTP V3 Alpha]
+
+The XMTP V3 SDKs are in **alpha** status and ready for you to start experimenting with.
+However, we do **not** recommend using alpha software in production apps. Software in this status will change as we add features and iterate based on feedback.
+
+
+XMTP V3 includes breaking changes from V2, most of which will be mitigated by the SDKs. Guidance about the migration path from V2 to V3 will be delivered in forthcoming XMTP V3 release notes.
+
+Join the [XMTP Devs 💪 group chat](https://converse.xyz/group-invite/e-KZyw77m-7sjUmEk5lgu) in Converse to keep up with the latest updates.
+
+:::
+
+With XMTP, you can enable real-time push notifications to keep users updated on new conversations and messages.
+
+For best practices, see [Best practices for push notifications](/notifications/notif-best-practices).
+
+### **Get a Welcome message topic ID**
```

Comment:
OK to include this section per Naomi's comment in the Notion draft - should this have a "coming soon"?